src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -179,6 +179,11 @@ extern id const kGRPCTrailersKey;
 @interface GRPCCall : GRXWriter
 
 /**
+ * Authority pseudo-header value.
+ */
+@property (atomic, readwrite) NSString *authority;
```

Comment:
Im guessing the name `authority` is too low level from a grpc abstraction perspective. The go driver calls it `ServerName`.